cosmos/operators/local.py

```diff
@@ -2,7 +2,6 @@
 
 import inspect
 import json
-import logging
```

Comment:
This was removed by pre-commit and I confirmed that it's not getting used in the module.